guides/04_Quality_Bandwidth_Connectivity.md

```diff
@@ -59,7 +49,7 @@ The Amazon Chime SDK for JavaScript produces several kinds of events on the [Aud
 >
 > You get the following WebRTC stats in the video stats widget:
 >
-> Upstream video information: Frame height, Frame width, Bitrate (bps), Packets Sent, Frame Rate (fps).
+> Upstream video information: Frame height, Frame width, Bitrate (bps), Packets Sent, Frame Rate (fps), Audio Jitter.
```

Comment:
Can we also add that if they want to get raw stats they can use the new parameter that @xuesichao added?

Reply:
Updated